.github/Dockerfile

```diff
@@ -13,6 +13,10 @@ ENV GIT_TAG=${GIT_TAG}
 ARG BUILD_OPTS=""
 ENV BUILD_OPTS=${BUILD_OPTS}
 
+# Add ncurses used for pretty output in bats tests
+RUN apk add --no-cache ncurses
+ENV TERM=linux
+
```

Comment:
For testing, okay, but this doesn't seem right as a final solution. It should really be added [pi-hole/docker-base-images#ftl-build/Dockerfile](https://github.com/pi-hole/docker-base-images/blob/master/ftl-build/Dockerfile) instead followed by a new tag.

Reply:
I know. This was just to test if ti works at all on CI.. see my comment below